src/transports/dash/manifest_parser.ts

```diff
@@ -71,16 +71,16 @@ export default function generateManifestParser(
     const { response, scheduleRequest } = args;
     const argClockOffset = args.externalClockOffset;
     const loaderURL = args.url;
-    const url = response.url == null ? loaderURL :
-                                       response.url;
+    const url = response.url === undefined ? loaderURL :
+                                             response.url;
     const data = typeof response.responseData === "string" ?
                    new DOMParser().parseFromString(response.responseData,
                                                    "text/xml") :
                    // TODO find a way to check if Document?
                    response.responseData as Document;
 
-    const externalClockOffset = serverTimeOffset == null ? argClockOffset :
-                                                           serverTimeOffset;
+    const externalClockOffset = serverTimeOffset === undefined ? argClockOffset :
```

Comment:
Why not :
`const externalClockOffset = serverTimeOffset ?? argClockOffset;`

Reply:
Done